market/connector/currencyConverter/openapi.json

```diff
@@ -0,0 +1,232 @@
+{
+  "openapi" : "3.0.0",
+  "info" : {
+    "description" : "Currency converter based on the Frankfurter.APP",
+    "version" : "1.0.2",
+    "title" : "CurrencyConverter"
+  },
+  "externalDocs" : {
+    "description" : "Find out more about Frankfurter",
+    "url" : "https://www.frankfurter.app/docs/"
+  },
+  "servers" : [ {
+    "url" : "https://api.frankfurter.app/"
+  } ],
+  "tags" : [ {
+    "name" : "currency"
+  } ],
+  "paths" : {
+    "/latest" : {
+      "get" : {
+        "tags" : [ "currency" ],
+        "summary" : "returns the latest rates",
+        "description" : "Multiple status values can be provided with comma separated strings",
+        "operationId" : "convertAmount",
+        "parameters" : [ {
+          "name" : "amount",
+          "in" : "query",
+          "required" : true,
+          "example" : 100,
+          "schema" : {
+            "type" : "number",
+            "format" : "double"
+          }
+        }, {
+          "name" : "from",
+          "in" : "query",
+          "description" : "quote against any Currency using the from parameter",
+          "required" : true,
+          "schema" : {
+            "type" : "string",
+            "enum" : [ "AUD", "BGN", "BRL", "CAD", "CHF", "CNY", "CZK", "DKK", "EUR", "GBP", "HKD", "HRK", "HUF", "IDR", "ILS", "INR", "ISK", "JPY", "KRW", "MXN", "MYR", "NOK", "NZD", "PHP", "PLN", "RON", "RUB", "SEK", "SGD", "THB", "TRY", "USD", "ZAR" ]
+          }
+        }, {
+          "name" : "to",
+          "in" : "query",
+          "description" : "limits returned rates to specified values",
+          "required" : false,
+          "schema" : {
```

Comment:
no we have the very same schema declared twice .. we could move it to /components and re-use it instead of duplicating it.